ssh/ssh.go

```diff
@@ -158,7 +158,7 @@ func (s *SshExec) ConnectAndExec(host string, commands []string, timeoutMinutes
 			if err != nil {
 				s.logger.LogError("Failed to run command [%v] on %v: Err[%v]: Stdout [%v]: Stderr [%v]",
 					command, host, err, b.String(), berr.String())
-				return nil, err
+				return nil, fmt.Errorf("%s", berr.String())
```

Comment:
That is nice! Thanks.  This helps with https://github.com/heketi/heketi/issues/171